src/src_V2/modules/manager/dashboard/index.jsx

```diff
@@ -1,7 +1,113 @@
+import {DatePicker} from "@mui/lab";
```

Comment:
@TigranYengiba04 jan, t think you prettier config is not working properly.

Reply:
done!

Reply:
great